components/segment-bridge/base/kustomization.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ./rbac
-- serviceaccount.yaml
+  - serviceaccount.yaml
```

Comment:
the file name here make it really hard to understand that it sets up the service account for reading namespaces on member clusters, and not only the account but also its cluster role and role binding as well as a role and role binding for reading it by particular members of the O11y team.

We should probably re-factor this to multiple files with clearer names at some point.

Reply:
out of scope, will be fixed in a separate change.